lib/terjira/presenters/common_presenter.rb

```diff
@@ -1,5 +1,6 @@
 require 'tty-table'
 require 'pastel'
+require 'unicode/display_width/string_ext'
```

Comment:
Implement `String#display_width` for L#52 (https://github.com/keepcosmos/terjira/pull/108/files#diff-5d863c726df4ed831ad90eb8c126bba58564ecfb6d8ae5889ab0686f58e51edfR53)